src/Documentation/UserManual/DeviceOpticalMarkerTracker.dox

```diff
@@ -0,0 +1,86 @@
+/*!
+\page DeviceOpticalMarkerTracker Optical Marker Tracker
+
+\section DeviceOpticalMarkerTrackerSupportedHwDevices Supported hardware devices
+
+- Any external webcam or laptop camera (forward or rear facing).
+
+\section DeviceOpticalMarkerTrackerSupportedPlatforms Included in packages
+
+- \ref PackageWin32
+- \ref PackageWin64
+- \ref PackageInterson
+- \ref PackageBk
+- \ref PackageStealthLink
+- \ref PackageThorLabs
+
+\section DeviceOpticalMarkerTrackerCalibration Calibration
+
+- Print the <a href="https://www.assembla.com/spaces/plus/documents/cd0kq6t7ir577cdmr6QqzO/download/cd0kq6t7ir577cdmr6QqzO">aruco calibration chessboard</a> and measure the side length of the markers in meters.
+- To generate a calibration file, run aruco_calibration.exe using the command
+  \code aruco_calibration.exe live:<camera-index> <output-file-name>.yml -size <marker-size-m> \endcode
+  where:
+  - <camera-index> is the integer index of the camera you want to use (usually 0 or 1).
+  - <output-file-name> is the desired output file name.
+  - <marker-size-mm> is the size of a marker side in meters.
+- With the video window selected:
+  - Press \b a to capture image and add to the calibration image set.
+  - Press \b c to run calibration on the captured images.  "repj" is a measure of the calibration error (normal is < 1).
+  - Press \b s to stop calibration.
+
+- For best results use at least 6 images including:
+  - At least one frontal image. \n
+    \image html ../images/DeviceOpticalMarkerTracker/Frontal.png
+  - Multiple images where the border of the chessboard is near the top or side image border. \n
+    \image html ../images/DeviceOpticalMarkerTracker/Edge.png
+  - Multiple images where the chessboard is held at an oblique angle to the camera. \n
+    \image html ../images/DeviceOpticalMarkerTracker/Oblique1.png
+    \image html ../images/DeviceOpticalMarkerTracker/Oblique2.png
+
+\section DeviceOpticalMarkerTrackerImages Creating images for printing
+- To print a \b single \b image, run aruco_print_marker.exe using the command
```

Comment:
Normally, targets are selected to be added to the installation package like this:
  INSTALL(
    TARGETS
      VolumeReconstructor
      CreateSliceModels
      DrawClipRegion
    DESTINATION "${PLUSLIB_BINARY_INSTALL}"
    COMPONENT RuntimeExecutables
    )

However, as Aruco's CMake configuration is messed up and it doesn't provide proper targets, you can either create the .exe targets manually, similarly to this:

# Import target "EditSequenceFile" for configuration "Release"
set_property(TARGET EditSequenceFile APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
set_target_properties(EditSequenceFile PROPERTIES
  IMPORTED_LOCATION_RELEASE "${_IMPORT_PREFIX}/bin/EditSequenceFile.exe"
  )

list(APPEND _IMPORT_CHECK_TARGETS EditSequenceFile )
list(APPEND _IMPORT_CHECK_FILES_FOR_EditSequenceFile "${_IMPORT_PREFIX}/bin/EditSequenceFile.exe" )

in PlusLibConfig.cmake.in; and then add this target to install similarly to this:

# Install
IF(PLUSAPP_INSTALL_BIN_DIR)
  INSTALL(TARGETS fCal 
    DESTINATION ${PLUSAPP_INSTALL_BIN_DIR}
    COMPONENT RuntimeExecutables
    )
ENDIF()

---

Or, you can try to add it to the package similarly to configuration and data files in InstallFiles.cmake. This may be slightly less nice solution.